ai-validator/.env.example

```diff
@@ -0,0 +1,8 @@
+AWS_REGION=<Region for default account>
+AWS_ACCOUNT_ID=<Account ID for default account>
```

Comment:
Could you explain what's the difference between AWS_ACCOUNT_ID and AUTH_ACCESS_ACCOUNT_ID and AUTH_ACCESS_ROLE_ID? Maybe change the names to make them more specific. It would be helpful if you can document how the two accounts work together (e.g. which account runs the demo, which account runs the tests, how the assume role works, what's the permissions required?)